playbooks/roles/dev-patcher/defaults/main.yml

```diff
@@ -8,4 +8,6 @@ dev_patcher_packages:
 #unrelated patches will fail (because not cloned)
 #conflicting patches will fail (because need resolving)
 dev_patcher_patches:
-  - 621206
+  - 637963
```

Comment:
can we have a comment above with the patch description so we dont have to go look them up?

Reply:
Done, I hope ansible handles this